reports/glides_terminal_missed_extraneous_predictions.livemd

```diff
@@ -318,6 +314,15 @@ filter_settings =
 Filter trip updates based on your settings.
```

Comment:
I'm not sure why I'm hitting this issue now (with this set of changes) but I'm trying to test out the updated extraneous/missed predictions report locally and hitting an issue on loading the data. It keeps hitting "aborted" with the following error message:

```
valuation process terminated - an exception was raised:
    ** (ArgumentError) errors were found at the given arguments:

  * 1st argument: not an already existing atom

        (erts 15.0) :erlang.binary_to_existing_atom("trip", :utf8)
        (atomic_map 0.9.3) lib/atomic_map.ex:53: AtomicMap.as_atom/3
        (atomic_map 0.9.3) lib/atomic_map.ex:21: anonymous fn/3 in AtomicMap.convert/2
        (stdlib 6.0) maps.erl:860: :maps.fold_1/4
        (atomic_map 0.9.3) lib/atomic_map.ex:22: anonymous fn/3 in AtomicMap.convert/2
        (stdlib 6.0) maps.erl:860: :maps.fold_1/4
        (elixir 1.17.2) lib/stream.ex:613: anonymous fn/4 in Stream.map/2
        (elixir 1.17.2) lib/enum.ex:4858: Enumerable.List.reduce/3
```

![Screenshot 2024-08-28 at 15-21-54 Glides Terminal Missed_Extraneous Predictions - Livebook](https://github.com/user-attachments/assets/dfbc34f3-4736-4e26-b032-6024009099ac)